guardian/prosemirror-elements
# feat: Cartoon element

## What does this change?
This adds a cartoon element with similar fields to the image element. The main difference is the cartoon element allows for multiple desktop and mobile images to be added and includes fields for basic styling when multiple images are present.

## How to test
You can run prosemirror-elements locally and see how the element works. Alternatively, there is a testing branch in composer (https://github.com/guardian/flexible-content/tree/bt/cartoon-element). Bear in mind, you will need to run `yalc` to pull in and depend on these changes (see this repo's README for details on how to do this). The work in Composer is still ongoing, so not everything on the test branch above will work exactly as expected.

## How can we measure success?
We have a cartoon element that we can build against in composer.

## Have we considered potential risks?
This PR contains minimal/no risk to existing operations. The cartoon element will be put behind a feature flag so can be thoroughly tested before release. There's a good chance this element will change (particular with regards to the data in/out transformers as we complete the work to support this element in composer.

## Images
<img width="720" alt="Screenshot 2022-12-20 at 09 30 31" src="https://user-images.githubusercontent.com/33927854/208632884-e74e3296-1bfb-467a-959b-dfeedd883758.png">
